src/CoreFoundation/Dispatch.cs

```diff
@@ -530,7 +535,11 @@ public DispatchQualityOfService GetQualityOfService (out int relative_priority)
 
 		[DllImport (Constants.libcLibrary)]
 		// dispatch_queue_t dispatch_get_global_queue (long priority, unsigned long flags);
-		extern static IntPtr dispatch_get_global_queue (nint priority, nuint flags);
+		extern static IntPtr dispatch_get_global_queue_priority (nint priority, nuint flags);
```

Comment:
As far as I can tell there's no function called `dispatch_get_global_queue_priority` in any of the platforms we support.